modules/custom/az_landing_page/az_landing_page.info.yml

```
@@ -0,0 +1,14 @@
+name: 'Quickstart Landing Page'
+type: module
+description: 'Provides a Landing Page content type similar to Quickstart Flexible Page.'
```

```suggestion
description: 'Provides a Landing Page content type similar to Quickstart Page.'
```
Ultimate nitpick: We don't refer to these as 'flexible' pages in QS2.